block_cascade/prefect/__init__.py

```diff
@@ -1,4 +1,10 @@
-from block_cascade.utils import PREFECT_VERSION
```

Comment:
Should we tell someone to install the `prefect` extra if they get here without prefect installed?